hotbar.coffee

```diff
@@ -47,6 +47,8 @@ class InventoryHotbarClient extends InventoryHotbarCommon
     @keys = game.plugins.get('voxel-keys') ? throw new Error('voxel-inventory-hotbar requires voxel-keys plugin')
     @wheelEnable = opts.wheelEnable ? false # enable scroll wheel to change slots?
     @wheelScale = opts.wheelScale ? 1.0  # mouse wheel scrolling sensitivity
+    @console = game.plugins.get('voxel-console')
+    @scrollLock = true; # allows scrolling on true
```

Comment:
Unnecessary semicolon; not required and should not normally be used in CoffeeScript